src/components/categoryslider/CategorySlider.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './CategorySlider.scss'
+
+
+function CategorySlider(products) {
+  return (
+    <div className='categoryslider-main-container'>
+      {
+        products && products?.products?.map((product)=>{
+          return(
+          <div key={product.id}>
+            <img src={product.image} alt="hi"/>
```

Comment:
alt text should be more meaningful towards the image

Reply:
+1

Reply:
Given meaningful alt for images